dotnet/src/Connectors/Connectors.AI.OpenAI/OpenAIRequestSettings.cs

```diff
@@ -0,0 +1,165 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Collections.Generic;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using Microsoft.SemanticKernel.AI;
+
+namespace Microsoft.SemanticKernel.Connectors.AI.OpenAI;
+
+/// <summary>
+/// Request settings for an OpenAI completion request.
+/// </summary>
+public class OpenAIRequestSettings : AIRequestSettings
+{
+    /// <summary>
+    /// Temperature controls the randomness of the completion.
+    /// The higher the temperature, the more random the completion.
+    /// </summary>
+    [JsonPropertyName("temperature")]
+    [JsonPropertyOrder(1)]
```

Comment:
No, this was in the original code. I'll remove this in a follow up PR because it makes extending this class problematic. I've created a task so I don't forget to do this.